node/resolvers/catalog/searchBreadcrumb.ts

```diff
@@ -99,21 +100,23 @@ export const resolvers = {
       mapArray,
       mapUnit,
       queryUnit,
+      categoryId = '',
     }: BreadcrumbParams) => {
       if (index === 0 && isCategoryMap(mapUnit)) {
-        return `/${queryUnit}/d`
+        return `/${queryUnit}/d/${categoryId}`
       }
-      if (index === 0 && isBrandMap(mapUnit)) {
-        return `/${queryUnit}/b`
+      else if (index === 0 && isBrandMap(mapUnit)) {
+        return `/${queryUnit}/b/${categoryId}`
       }
-      if (mapArray.every(isCategoryMap)) {
-        return `/${sliceAndJoin(queryArray, index + 1, '/')}`
+      else if (mapArray.every(isCategoryMap)) {
+        return `/${sliceAndJoin(queryArray, index + 1, '/')}/${mapUnit}/${categoryId}`
+      } else {
+        return `/${sliceAndJoin(queryArray, index + 1, '/')}/${mapUnit}/${categoryId}?map=${sliceAndJoin(
```

Comment:
in this part we may be returning anything and not just cateogry, like other maps, for example, sellerids, productClusterIds, is category Id relevant in this case?